scripts/setup-macos.sh

```diff
@@ -94,7 +94,7 @@ function install_fmt {
 function install_folly {
   github_checkout facebook/folly "v2022.11.14.00"
   OPENSSL_ROOT_DIR=$(brew --prefix openssl@1.1) \
-    cmake_install -DBUILD_TESTS=OFF
+  cmake_install -DBUILD_TESTS=OFF -DFOLLY_HAVE_INT128_T=ON
```

Comment:
This needs to be added to the other install scripts as well!

Reply:
done. Thanks!

Reply:
The linux wheels (and any not running the script during the run) will fail as long as the docker image is not rebuild

Reply:
I opened https://github.com/facebookincubator/velox/issues/4864 to rebuild the docker image.